graphql-cli/src/main/java/io/ballerina/graphql/generator/CodeGenerator.java

```diff
@@ -158,6 +162,13 @@ private void generateUtils(String projectName, AuthConfig authConfig, List<SrcFi
                 UTILS_FILE_NAME, utilSrc));
     }
 
+    private void generateConfigTypes(String projectName, AuthConfig authConfig, List<SrcFilePojo> sourceFiles)
+            throws ConfigTypesGernerationException {
+        String utilSrc = ConfigGenerator.getInstance().generateSrc(authConfig);
```

Comment:
Let's rename this variable as `configTypesSrc`

Reply:
Renamed in https://github.com/ballerina-platform/graphql-tools/commit/127efce1497eae87b00a2136832a9afc7dd0e3ef